portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,18 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h3>Hello Andrew!</h3>");
+    String[] ArrayList = {"Hey!", "Check", "here!"};
+    String To_Json = convertToJson_Gson(ArrayList);
```

Comment:
You can refer to the open-sourced Google Java style guide for more thorough reference, but in general, in Java, we would use camel case, e.g., "toJson".

The same also applies to method names, e.g., prefer convertToJson(). About the name part "Gson" in the method name, you could keep it like "convertToJsonWithGson" if you think using Gson is something particularly important for the caller to be aware of; otherwise, prefer a shorter name without exposing the underlying implementation details with "convertToJson".

In general, we push for a consistent style guide to make it easy for everyone to read the code more easily. This is especially important when you work with a big team. Also, code is read way more often than written.